package.json

```diff
@@ -90,6 +90,7 @@
     "@openzeppelin/contracts": "4.4.2",
     "@sentry/react": "^6.10.0",
     "@sentry/tracing": "^6.10.0",
+    "@truffle/hdwallet-provider": "^2.0.8",
```

Comment:
This should probably go to devDependencies

Reply:
no doubt